src/NLog.Web.AspNetCore/README.md

```diff
@@ -0,0 +1,35 @@
+# NLog.Web.AspNetCore
+
+- LoggerProvider for ASP.NET Core platform via extension methods for ILoggingProvider, IWebHostBuilder, and IHostBuilder.
+- Log all properties of HttpContext and IHostEnvironment via [layout renderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore) just by changing the NLog.config file.
```

Comment:
"Extends logging output with details from the active HttpContext via layout renderers, just by changing NLog.config file."

Reply:
Completed